test/tasks/task.spec.ts

```diff
@@ -0,0 +1,125 @@
+import test from 'ava';
+import {spy} from 'sinon';
+
+import {
+  taskMiddleware,
+  withTask,
+  taskCreator,
+  drainTasksForTesting,
+  Task,
+} from '../../src/tasks';
+
+import {
+  taskStore,
+  SET_TASK_SYNC,
+  SET_TASK_ASYNC
+} from './fixtures';
+
+// tasks
+const xhrHandlerSpy = spy();
+const XHR_TASK = taskCreator(xhrHandlerSpy, 'XHR_TASK');
+
+const ADD = payload => ({type: ADD, payload});
+const SET_SYNC = payload => ({type: SET_SYNC, payload});
+const SET_ASYNC = payload => ({type: SET_ASYNC, payload});
+const SET_SUCCESS = payload => ({type: SET_SUCCESS, payload});
+const BAD = () => ({type: BAD, payload: {}});
+
+export const reducer = (state = 0, action) => {
+  const {payload} = action;
+
+  switch (action.type) {
```

Comment:
In general, yes, but I prefer not to in tests. Better to test just one thing at a time.